tests/unit/acquisition/test_optimizer.py

```diff
@@ -74,6 +75,7 @@ def test_optimize_continuous(
 
 
 @random_seed
+@pytest.mark.parametrize("batchify_type", [batchify, batchify_greedy])
```

Comment:
what differentiates `batchify_greedy` from `batchify`, and can we have a test for that?